### PR TITLE
Clean up OSGi use of CDIService

### PR DESCRIPTION
**Pull request by benjamin-confino**
Follow up to #27377

Another place where using the new helper class will make our code better.

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.internal.common/transformed.bnd [hunk 1]
```diff
@@ -29,5 +29,4 @@ Private-Package: \
 
 -dsannotations: \
 	io.openliberty.microprofile.telemetry.internal.common.info.OpenTelemetryInfoFactoryImpl,\
-	io.openliberty.microprofile.telemetry.internal.common.rest.jakarta.TelemetryClientAsyncTaskWrapper,\
-	io.openliberty.microprofile.telemetry.internal.interfaces.OpenTelemetryAccessor
+	io.openliberty.microprofile.telemetry.internal.common.rest.jakarta.TelemetryClientAsyncTaskWrapper
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.internal.common/transformed.bnd:32]: Copyright date on this file needs updating.

File: dev/io.openliberty.microprofile.telemetry.internal.common/original.bnd [hunk 2]
```diff
@@ -28,5 +28,4 @@ Private-Package: \
 
 
 -dsannotations: \
-	io.openliberty.microprofile.telemetry.internal.common.info.OpenTelemetryInfoFactoryImpl,\
-	io.openliberty.microprofile.telemetry.internal.interfaces.OpenTelemetryAccessor
+	io.openliberty.microprofile.telemetry.internal.common.info.OpenTelemetryInfoFactoryImpl
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.internal.common/original.bnd:31]: Copyright date on this file needs updating.

**Comment by Azquelt** [comment]
This looks really good, other than the two files which need the copyright date updated.

**Comment by benjamin-confino** [comment]
#libby

**Review by Azquelt — APPROVED** [review]
Looks great, thanks.

**Pull request merged**